src/languages/fortran.coffee

```diff
@@ -18,6 +18,10 @@ module.exports = {
     "F90"
     "f95"
     "F95"
+    "f03"
+    "F03"
+    "f08"
+    "F08"
```

Comment:
Please run `npm run docs` and commit the changes. This will update the documentation including the Language section of the README: https://github.com/Glavin001/atom-beautify#language-support